scripts/bootstrap-ansible.sh

```diff
@@ -2,4 +2,4 @@
 
 # minimal bootstrapping before kicking off ansible:
 #  install only what ansible needs to survive, or doesn't know how to do
-sudo dnf -y install python2 python2-dnf libselinux-python
+sudo yum -y install python2 python2-yum libselinux-python
```

Comment:
This package does not exist.

Reply:
Seems like the command never errored out (or had negative effects) so I never noticed.  Box seems to work fine anyways, but I'll push a new commit.